iamctl/pkg/utils/resourceProperties.go

```diff
@@ -81,7 +104,9 @@ func AreSecretsExcluded(resourceConfigs map[string]interface{}) bool {
 	if secretsExcluded, ok := resourceConfigs[EXCLUDE_SECRETS_CONFIG].(bool); ok {
 		return secretsExcluded
 	}
-	return true
+
+	// Check if secrets are excluded for all resources.
+	return TOOL_CONFIGS.ExcludeSecrets
```

Comment:
What is the behaviour if EXCLUDE_SECRETS config (global) is not set by the user?